//how to access the array in error function 

const resultSum = <T,>(value:T[]):T => {
    const sums = 4;
    return value[sums];
}

